csrc/fusion_profiler.cpp

```diff
@@ -675,15 +749,21 @@ void FusionProfiler::stop() {
           fp->segments_[kp_idx].state() == ProfilerState::Finished,
           "SegmentProfiler ProfilerState is not Finished!",
           fp->segments_[kp_idx].state());
+      kprof.segment_id = static_cast<size_t>(kp_idx);
       kprof.input_bytes = segment(kp_idx).inputBytes();
       kprof.output_bytes = segment(kp_idx).outputBytes();
-      kprof.effective_bandwidth_gbs =
-          (double)(kprof.input_bytes + kprof.output_bytes) / kprof.time_ms *
-          mb_divider;
+      kprof.effective_bandwidth_gbs = (double)(segment(kp_idx).inputBytes() +
```

Comment:
nitpick, what's the reason for code change here? wasn't `segment(kp_idx).inputBytes()` exactly what we assigned to `kprof.input_bytes` earlier?

Reply:
I have no idea why I changed this.  I changed it back.